src/main/resources/com/google/api/codegen/py/standalone_sample.snip

```diff
@@ -23,6 +23,9 @@
   @# [START {@sample.regionTag}]
 
   {@importList(sample.initCode.importSection.appImports)}
+  @if convertText(sample)
```

Comment:
Can you assign this to a template variable around line 8, so you're not calling the function twice?
I think I saw in some of the templates that you can assign more than one variable in  a `@let` statement by separating them with whitespace  / slash-newline